src/daemon/ca/certauth.rs

```diff
@@ -102,17 +89,11 @@ pub struct CertAuth {
     children: HashMap<ChildHandle, ChildDetails>,
     routes: Routes,
 
-    #[serde(skip_serializing_if = "Rtas::is_empty", default = "Rtas::default")]
+    #[serde(skip_serializing_if = "Rtas::is_empty", default)]
     rtas: Rtas,
 
-    #[serde(
-        skip_serializing_if = "AspaDefinitions::is_empty",
-        default = "AspaDefinitions::default"
-    )]
+    #[serde(skip_serializing_if = "AspaDefinitions::is_empty", default)]
     aspas: AspaDefinitions,
-
-    #[serde(skip_serializing, default = "CertAuthStatus::default")]
-    status: CertAuthStatus,
```

Comment:
What was the motivation for removing this field entirely? Is it unused?

Reply:
It is unused. It was there earlier because earlier versions had this and it could be present in snapshots. But snapshots are rebuilt in the migration to 0.9.x so this is now safe to remove completely.